adaptive_navigation/lib/scaffold.dart

```diff
@@ -380,10 +381,6 @@ class AdaptiveNavigationScaffold extends StatelessWidget {
   }
 }
 
-bool _isLargeScreen(BuildContext context) {
-  return MediaQuery.of(context).size.width > 960.0;
-}
+bool _isLargeScreen(BuildContext context) => getWindowType(context) >= AdaptiveWindowType.l;
 
-bool _isMediumScreen(BuildContext context) {
-  return MediaQuery.of(context).size.width > 640.0;
-}
+bool _isMediumScreen(BuildContext context) => getWindowType(context) == AdaptiveWindowType.m;
```

Comment:
should this be <= AdaptiveWindowType.m instead of == ?

Reply:
Okay